src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started digging");

```

Comment:
The message 'Excavator started digging' could be more informative by specifying that it is the Excavator machine that is working. According to the checklist, we should use informative messages that indicate the type of machine.